### PR TITLE
[css/en] Add media queries section, demonstrating basics of print and mobile

### PR DESCRIPTION
**Pull request by Glutnix**
Also improved link to Can I Use
Change HTML snippets to HTML 5
Fix minor whitespace issues
Add myself as a contributor

### CODE REVIEW

**Comment by vendethiel** [comment]
Sorry for the delay, it seems there are some merge conflicts now. Could you fix them?

**Comment by Glutnix** [comment]
@vendethiel Merged and fixed.

File: css.html.markdown [hunk 1]
```diff
@@ -252,6 +252,48 @@ recommended that you avoid its usage.
 * `B` is next.
 * `D` is the last one.
 
+## Media Queries
+
+CSS Media Queries are a feature in CSS 3 which allows you to specify when certain CSS rules should be applied, such as when printed, or when on a screen with certain dimensions or pixel density.
```
Comment (geoffliu) [inline css.html.markdown]: Could we add a note that media queries do not add to the specificity? That was a bit surprising, to me at least.

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**